### PR TITLE
remove lots of things, add header ui, add home page content, no funct…

### PR DESCRIPTION
**Pull request by immortaldogg**
remove lots of things, add header ui, add home page content, no functionality yet

### CODE REVIEW

**Comment by immortaldogg** [comment]
Merge to work on mini-challenge-2, might have to improve later

**Comment by qtd-wizeline** [comment]
No responsive UI for mobile view, we can use `@media` for each screen size

**Comment by qtd-wizeline** [comment]
Should separate `styled components` and `logic components` into  different files, for example:

```
Components
|--App
|--Header
    |--index.js
    |--Header.js // logic
    |--Header.styled.js // styled
    |--Header.test.js // test
|--...
```

Then we can import styled component into logic component, which shouldn't bother us if the components grow bigger in the future

File: src/index.jsx [hunk 1]
```diff
@@ -1,7 +1,7 @@
 import React from 'react';
 import ReactDOM from 'react-dom';
 
-import App from './components/App';
+import App from './components/App/App';
```
Comment (qtd-wizeline) [inline src/index.jsx:4]: For readability, in `./components/App/` folder, we could create file `index.js` with content:
 ```js
 export { default } from './App';
 ```
By doing this 
```js
import App from './components/App/App';
```
becomes
 
```js
import App from './components/App';
```

Similarly for `import` in other files

**Pull request merged**